azbfs/url_file.go

```diff
@@ -17,6 +17,16 @@ type FileURL struct {
 	path           string
 }
 
+//rename me?
```

Comment:
Minor: Our convention is to write things-to-do in the form "TODO" followed by action. In cases like this, can use the word "consider" in the text. E.g.:
// TODO: consider renaming me